README.md

```diff
@@ -1,37 +1,51 @@
 ![Deploy](https://github.com/DFE-Digital/govuk-rails-boilerplate/workflows/Deploy/badge.svg)
 
-# Tim's secret Spike
+# EYFS Reform Spike
 
 ## Getting started on Docker
 
 On Mac OS, [Docker Desktop / Docker for Mac](https://docs.docker.com/docker-for-mac/install/)
-will need to be installed first
+will need to be installed first and running
 
 Setup a `.env` file to hold environment variables and fill in the missing values
 (do not commit this file)
 
 `cp .env.example .env`
 
+add development database to .env: govuk_rails_boilerplate_development
+
+### To run the application locally with docker:
+
 NOTE: Ensure no instances of PostgreSQL are running in the background as this can cause conflicts when attempting to run the docker instance of PostgreSQL
 If PostgreSQL was installed using homebrew, in the terminal use:
 1. `brew services` - to check for any running instances,
 2. `brew services stop postgres`
 
-Add the following lines at the end of the `docker-compose.yml` file (if not already present):
-
-`environment:`
-      `POSTGRES_PASSWORD: [REPLACE WITH THE DATABASE_PASSWORD IN .env FILE]`
-
 To run the application locally with docker:
 
 `docker-compose build && docker-compose up`
 
-To setup the database running in a docker container:
+If docker has been setup correctly, running `sudo docker ps`, you should see 2 containers running like this:
+
+CONTAINER ID   IMAGE                   COMMAND                  CREATED       STATUS         PORTS                    NAMES
```

Comment:
Can this be wrapped in a code block so it reads nicer

Reply:
done ill push it